File: Backend/App/models/stock.py
from sqlalchemy import Column,String,Integer,Boolean,ForeignKey
from Backend.App.core.database import Base
from Backend.App.models.medicine import Medicine

class Stock(Base):
    __tablename__ ="Stock"

    StockId = Column(Integer,primary_key=True,index=True)
    MedicineId = Column(Integer,ForeignKey(Medicine.MedicineId))
    Quantity = Column(Integer)

File: DotNetBackend/ViveksHomoeoClinic/ViveksHomoeoClinic/Models/EFModel/Stock.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ViveksHomoeoClinic.Models.EFModel
{
    public class Stock
    {
        [Key]
        public int StockId { get; set; }
        [Required]
        public int MedicineId { get; set; }

        [ForeignKey(nameof(MedicineId))]
        public Medicine Medicine { get; set; }
        public int Quantity { get; set; }
    }
}
